js/slouchdetection.js

```diff
@@ -4,16 +4,16 @@
 
 const LOWERED_FACE_DISTANCE = 30;
 const TOO_BIG_FACE_SIZE = 200;
-const SLOUCHOPTIONS = {
-    "forwardLeaningSlouch": 1,
-    "slouchDown": 2,
-    "none": false
+const SLOUCH_ATTRIBUTES = {
+    "FORWARD_LEANING_SLOUCH": 0,
+    "SLOUCH_DOWN": 1,
+    "NONE": 2
 };
 
 class SlouchDetectEventHandler extends CameraControllerEventHandler {
     onFrame(frame, tab, faces) {
         if (faces.length > 0) {
-            if (slouchDetect(faces) === 1) {
+            if (slouchDetect(faces) === 0) {
```

Comment:
Compare to the enum value instead of a magic number. Also consider using a switch statement.